### PR TITLE
Release liquibase-cdi-jakarta to maven repositories

### PR DESCRIPTION
**Pull request by DCCSKrezovic**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [x] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Description

This updates the github actions to include the newly added liquibase-cdi-jakarta module.

Fixes [#3962](https://github.com/liquibase/liquibase/issues/3962)

### CODE REVIEW

**Comment by nvoxland** [comment]
**Initial/Pre-Review Thoughts**

Thanks for the updates to the release logic. 

Questions I have:
- Have we looked at the pom.xml that gets distributed in the liquibase-cdi-jakarta.jar to make sure it's what users need/want and not build-related stuff in there? 
- Are we happy with the maven coordinates before the first release? 
  - `liquibase-cdi-jakarta` seems better than `liquibase-jakarta-cdi`? (I think so). 
  - Is it better or worse to use "jakarta" in the version for the same `liquibase-cdi` name? (I'm not sure off hand)

Potential risks:
- Can't really test it until release time

What could make the full review difficult:
- Nothing

**Comment by DCCSKrezovic** [comment]
Hi @nvoxland, thanks for the review.

- It's exactly the same pom.xml as for liquibase-cdi (non-jakarta), except the Jakarta-specific dependencies, obviously. So, the final dependency here is just the cdi-api (provided) and liquibase-core (required)
- The naming has been agreed and discussed in https://github.com/liquibase/liquibase/pull/3642

**Review by filipelautert — APPROVED** [review]
In 4.20.0 we added the new module. Now we need to release it, thanks for the PR @DCCSKrezovic !

**Review by suryaaki2 — APPROVED** [review]

**Comment by XDelphiGrl** [comment]
hi, @filipelautert! The liquibase-artifacts.zip from this PR's build do not contain the renamed CDI. Is that expected?

**Contents of liquibase-artifacts.zip**
image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -412,7 +412,10 @@ jobs:
           cp liquibase-cdi/target/liquibase-cdi-0-SNAPSHOT.jar artifacts
           cp liquibase-cdi/target/liquibase-cdi-0-SNAPSHOT-sources.jar artifacts
           cp liquibase-cdi/target/liquibase-cdi-0-SNAPSHOT-javadoc.jar artifacts
-          
+
+          cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT.jar artifacts
+          cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT-sources.jar artifacts
+          cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT-javadoc.jar artifacts
```
Comment (filipelautert) [inline .github/workflows/build.yml:418]: Hello @XDelphiGrl - the copy commands are here , but as we execute the scripts from master they will only run when this pr is merged.

Comment (XDelphiGrl) [reply .github/workflows/build.yml:418]: Thank you for the explanation, @filipelautert. I'll go ahead and do my code review and we can get this merged. I'll check that the build produces the correctly named artifacts.

**Review by XDelphiGrl — APPROVED** [review]
Coming soon to a Maven near you ... liquibase-cdi-jakarta! This PR updates build logic to for the CDI artifacts. The sign-artifacts.sh script generates the MD5 and SHA1at the directory level; the renamed CDI artifacts will go through the same signing process as the previous CDI artifacts.
* The functionality of these build changes cannot be validated until they are merged to master.
* QA will check the generated artifacts from the concomitant master build for correctness.
* Test harness and functional tests do not exercise CDI; results unnecessary. 

APPROVED

**Pull request merged**